scripts/zfs-helpers.sh

```diff
@@ -165,6 +166,16 @@ if [ "${INSTALL}" = "yes" ]; then
 	    "$INSTALL_UDEV_RULE_DIR/90-zfs.rules"
 	install "$CMD_DIR/zpool/zpool.d" \
 	    "$INSTALL_SYSCONF_DIR/zfs/zpool.d"
+	install "$CONTRIB_DIR/pyzfs/libzfs_core" \
+	    "$INSTALL_PYTHON_DIR/libzfs_core"
+	# Ideally we would install these in the configured ${libdir}, which is
```

Comment:
This is, unfortunately, the only way i managed to run pyzfs ztests in-tree: "libzfs_core" code coverage is now at ~97% according to codecov, which i think is good compared to the decreased (!?) coverage (-0.2%) introduced by the zpool split tests.